pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -216,6 +219,10 @@ let
       ln -s ${stdenv.cc}/bin/clang              third_party/llvm-build/Release+Asserts/bin/clang
       ln -s ${stdenv.cc}/bin/clang++            third_party/llvm-build/Release+Asserts/bin/clang++
       ln -s ${llvmPackages.llvm}/bin/llvm-ar    third_party/llvm-build/Release+Asserts/bin/llvm-ar
+    '' + optionalString useOzone ''
+       #substituteInPlace third_party/wayland/features.gni \
+       # --replace 'system_wayland_scanner_path = "/usr/bin/wayland-scanner"' \
+       # 'system_wayland_scanner_path = "${wayland}/bin/wayland-scanner"'
```

Comment:
And please don't forget to remove this, if `system_wayland_scanner_path = "${wayland}/bin/wayland-scanner";` works as expected - thanks :)